pkg/ucp/frontend/controller/credentials/azure/deleteazurecredential_test.go

```diff
@@ -132,7 +132,7 @@ func setupCredentialMocks(mockStorageClient store.MockStorageClient) {
 	datamodelCredential := datamodel.AzureCredential{
 		BaseResource: v1.BaseResource{},
 		Properties: &datamodel.AzureCredentialResourceProperties{
-			Kind: datamodel.AWSCredentialKind,
+			Kind: datamodel.AWSAccessKeyCredentialKind,
```

Comment:
I know this is unrelated to your PR, but do you know why we are testing with AWS kind for Azure credentials?

Reply:
thats interesting.. looks like a mistake. Will fix this.

Reply:
chnaged to datamodel.AzureWorkloadIdentityCredentialKind